cpp/cmake/deps/concurrentqueue.cmake

```diff
@@ -0,0 +1,36 @@
+#
```

Comment:
Is `concurrentqueue` being used (I didn't see where it is)? If so, I think we need a `LICENSE.concurrentqueue`. If not, then we can remove this cmake file.

Reply:
Thanks @grlee77 ! `concurrentqueue` is not used. let me remove the cmake file.

Reply:
Removed it in the latest patchset.